public/assets/js/dubtrack/src/views/room/player.view.js

```diff
@@ -75,6 +61,54 @@ Dubtrack.View.Player = Backbone.View.extend({
 		this.fetchSong();
 	},
 
+	setPlayer : function(player_object){
+		if(this.player_instance) return;
```

Comment:
Identifier 'player_instance' is not in camel case.<br>Expected '{' and instead saw 'return'.